lib/widgets/fields/alignment_field.dart

```diff
@@ -0,0 +1,186 @@
+                        }
+                      });
+                    },
+                    isSelected: selectedOption == option,
+                  );
+                }),
+              );
+            }),
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class AlignmentFieldItem extends StatefulWidget {
+
+  final AlignmentOption option;
+  final void Function(Alignment) onChanged;
+  final bool isSelected;
+
+  const AlignmentFieldItem({
+    super.key,
+    required this.onChanged,
+    required this.isSelected,
+    required this.option,
+  });
+
+  @override
+  State<AlignmentFieldItem> createState() => _AlignmentFieldItemState();
+}
+
+class _AlignmentFieldItemState extends State<AlignmentFieldItem> {
+
+  BorderRadius? borderRadius;
+  String text = "";
+  Alignment? alignment;
+
+  AlignmentOption? onHoverOption;
+  bool isHover = false;
+
+  @override
+  Widget build(BuildContext context) {
+    switch (widget.option) {
+      case AlignmentOption.topLeft:
+        borderRadius = const BorderRadius.only(topLeft: Radius.circular(7));
+        text = "Top Left";
+        alignment = Alignment.topLeft;
+        break;
+      case AlignmentOption.topCenter:
+        text = "Top";
+        alignment = Alignment.topCenter;
+        break;
+      case AlignmentOption.topRight:
+        borderRadius = const BorderRadius.only(topRight: Radius.circular(7));
+        text = "Top Right";
+        alignment = Alignment.topRight;
+        break;
+      case AlignmentOption.centerLeft:
+        text = "Left";
+        alignment = Alignment.centerLeft;
+        break;
+      case AlignmentOption.center:
+        text = "Center";
+        alignment = Alignment.center;
+        break;
+      case AlignmentOption.centerRight:
+        text = "Right";
+        alignment = Alignment.centerRight;
+        break;
+      case AlignmentOption.bottomLeft:
+        borderRadius = const BorderRadius.only(bottomLeft: Radius.circular(7));
+        text = "Bottom Left";
+        alignment = Alignment.bottomLeft;
+        break;
+      case AlignmentOption.bottomCenter:
+        text = "Bottom";
+        alignment = Alignment.bottomCenter;
+        break;
+      case AlignmentOption.bottomRight:
+        borderRadius = const BorderRadius.only(bottomRight: Radius.circular(7));
+        text = "Bottom Right";
+        alignment = Alignment.bottomRight;
+        break;
+      default:
+        borderRadius = BorderRadius.circular(0);
+        alignment = Alignment.center;
+        text = "";
+        break;
+    }
+
+    return InkWell(
+      onTap: () {
+        setState(() {
+          widget.onChanged(alignment!);
+        });
+      },
+      onHover: (val) {
+        setState(() {
+          isHover = val;
+          onHoverOption = widget.option;
+        });
+      },
+      child: Container(
+        padding: const EdgeInsets.only(bottom: 2),
+        margin: const EdgeInsets.all(0),
+        alignment: Alignment.center,
+        width: 105.3,
```

Comment:
For our widgets, width is dependent on the width of property drawer. If in future, we changed the width of property drawer, or it may appear differently on mobile or tablet, then all the widgets should automatically comply with such width change.
So, we need to make this widget in a way that it divides the available space in three equal sizes which can easily be done used Expanded widget. 
The height can be fixed.